### ISSUE TITLE
Output video file is missing audio decryption

### ISSUE THREAD

**Issue by DarkHorse-APP2** [glomatico/gamdl#7]
For some reason while downloading videos files, output file only have video track decrypted but audio track is still encrypted. Seems like is due MP4Box that is not able to decrypt all the audio segments properly.

**Comment by glomatico** [comment]
Hi! Could you send the link for the video? Also, MP4Box is not responsible for decryption, mp4decrypt is.

**Comment by DarkHorse-APP2** [comment]
> Hi! Could you send the link for the video? Also, mp4decrypt is responsible for decryption, not MPBox.

Yeah I think you're right, trying decryption with shacka packager works fine. Here's the video link: https://music.apple.com/us/music-video/treehouse-feat-shotty-horroh/1479402610

**Comment by glomatico** [comment]
I was not able to reproduce your issue. Both audio and video plays fine.
